docusaurus/docs/recipes/redux.mdx

```diff
@@ -0,0 +1,407 @@
+---
+id: redux
+title: Setting up Redux
+sidebar_label: Redux
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={ useBaseUrl('recipes-assets/setting-up-redux/redux-logo.svg') } alt="Redux" width="300" />
+
+In this recipe, you will be guided through the process of setting up a disk persistance-enabled Redux store in your React Native app.
+
+> ℹ️ It is assumed you have understanding of Redux concepts, how it works and knowledge of the most common tools revolving Redux's ecossystem. In case you don't, first please go through the [official introductory document](https://redux.js.org/basics/basic-tutorial) before proceeding. We also recommend you to also read the Advanced Tutorial material, otherwise it might be difficult for you to follow this recipe.
```

Comment:
```suggestion
> ℹ️ It is assumed you have an understanding of Redux concepts, how it works, and knowledge of the most common tools revolving Redux's ecosystem. In case you don't, first please go through the [official introductory document](https://redux.js.org/basics/basic-tutorial) before proceeding. We also recommend you read the Advanced Tutorial material, otherwise, it might be difficult for you to follow this recipe.
```

Reply:
Done ✅